raiden/tests/scenarios/bf2_long_running.yaml

```diff
@@ -0,0 +1,335 @@
+      - serial:
+          name: "Payment from node0 to node3"
+          tasks:
+            - transfer: {from: 0, to: 3, amount: 200_000_000_000_000_000, lock_timeout: 30}
+      - wait: 100
+      - parallel:
+          name: "Assert channels after transfers"
+          tasks:
+            - assert: {from: 2, to: 3, total_deposit: 101_000_000_000_000_000, balance: 0, state: "opened"}
+            - assert: {from: 3, to: 2, total_deposit: 101_000_000_000_000_000, balance: 202_000_000_000_000_000, state: "opened"}
+            - assert: {from: 0, to: 1, total_deposit: 260_000_000_000_000_000, balance: 0, state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 100_000_000_000_000_000, balance: 360_000_000_000_000_000, state: "opened"}
+            - assert: {from: 1, to: 2, total_deposit: 100_000_000_000_000_000, balance: 0, state: "opened"}
+            - assert: {from: 2, to: 1, total_deposit: 100_000_000_000_000_000, balance: 200_000_000_000_000_000, state: "opened"}
+      - serial:
+          name: "Starting closing edge cases"
+          tasks:
+            # node2 tries to make a deposit to the channel that is being closed (fail 409)
+            - deposit: {from: 2, to: 4, total_deposit: 200_000_000_000_000_000, expected_http_status: 409}
+            - wait: 100
+            # node2 sends tokens to node1
+            - transfer: {from: 2, to: 1, amount: 10_000_000_000_000_000, lock_timeout: 30}
+            - wait: 100
+            - assert: {from: 1, to: 2, total_deposit: 100_000_000_000_000_000, balance: 10_000_000_000_000_000, state: "opened"}
+            - assert: {from: 2, to: 1, total_deposit: 100_000_000_000_000_000, balance: 189_000_000_000_000_000, state: "opened"}
+            # node1 sends tokens to node0
+            - transfer: {from: 1, to: 0, amount: 10_000_000_000_000_000, lock_timeout: 30}
+            - wait: 100
+            - assert: {from: 0, to: 1, total_deposit: 260_000_000_000_000_000, balance: 10_000_000_000_000_000, state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 100_000_000_000_000_000, balance: 299_000_000_000_000_000, state: "opened"}
+            # node4 sends tokens to node2 (0 -> 1-> 2)
+            - transfer: {from: 4, to: 2, amount: 10_000_000_000_000_000, lock_timeout: 30}
+            - wait: 100
+      - parallel:
+          name: "Assert for the channel states after mediated transfers were completed"
+          tasks:
+            - assert: {from: 1, to: 2, total_deposit: 100_000_000_000_000_000, balance: 0, state: "opened"}
+            - assert: {from: 2, to: 1, total_deposit: 100_000_000_000_000_000, balance: 200_000_000_000_000_000, state: "opened"}
+            - assert: {from: 0, to: 1, total_deposit: 260_000_000_000_000_000, balance: 0, state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 100_000_000_000_000_000, balance: 309_000_000_000_000_000, state: "opened"}
```

Comment:
Can you please go through all these asserts and make sure the calculations are correct?

Reply:
Done